movies-api/utils/middleware/validationHandler.js

```diff
@@ -2,11 +2,11 @@ const boom = require('@hapi/boom');
 const joi = require('@hapi/joi');
 
 function validate(data, schema) {
-  const { error } = joi.validate(data, schema);
+  const { error } = joi.object(schema).validate(data);
```

Comment:
Si esto funciona con la nueva version de Joi, entonces el PR también debería actualizar el package.json y yarn.lock

Reply:
Listo actualice el package.json and package-lock.json con la nueva version de @hapi/joi 🙊